Formula/kat.rb

```diff
@@ -14,31 +14,53 @@ class Kat < Formula
 
   head do
     url "https://github.com/TGAC/KAT.git"
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
   end
 
   needs :cxx11
 
   depends_on "pkg-config" => :build
-  depends_on "boost"
-  depends_on "python3"
+  depends_on "numpy"
+  depends_on "scipy"
+  depends_on "python"
 
-  def install
-    # Reduce memory usage for Circle CI.
-    ENV["MAKEFLAGS"] = "-j8" if ENV["CIRCLECI"]
+  resource "tabulate" do
+    url "https://files.pythonhosted.org/packages/12/c2/11d6845db5edf1295bc08b2f488cf5937806586afe42936c3f34c097ebdc/tabulate-0.8.2.tar.gz"
+    sha256 "e4ca13f26d0a6be2a2915428dc21e732f1e44dad7f76d7030b2ef1ec251cf7f2"
+  end
+
+  resource "matplotlib" do
+    url "https://files.pythonhosted.org/packages/6c/90/cf10bb2020d2811da811a49601f6eafcda022c6ccd296fd05aba093dee96/matplotlib-2.1.0.tar.gz"
+    sha256 "4b5f16c9cefde553ea79975305dcaa67c8e13d927b6e55aa14b4a8d867e25387"
+  end
+
+  resource "pyparsing" do
+    url "https://files.pythonhosted.org/packages/3c/ec/a94f8cf7274ea60b5413df054f82a8980523efd712ec55a59e7c3357cf7c/pyparsing-2.2.0.tar.gz"
+    sha256 "0832bcf47acd283788593e7a0f542407bd9550a55a8a8435214a1960e04bcb04"
+  end
 
-    ENV["PYTHON_EXTRA_LDFLAGS"] = "-undefined dynamic_lookup"
+  resource "python-dateutil" do
+    url "https://files.pythonhosted.org/packages/54/bb/f1db86504f7a49e1d9b9301531181b00a1c7325dc85a29160ee3eaa73a54/python-dateutil-2.6.1.tar.gz"
+    sha256 "891c38b2a02f5bb1be3e4793866c8df49c7d19baabf9c1bad62547e0b4866aca"
+  end
+
+  resource "pytz" do
+    url "https://files.pythonhosted.org/packages/a4/09/c47e57fc9c7062b4e83b075d418800d322caa87ec0ac21e6308bd3a2d519/pytz-2017.2.zip"
+    sha256 "f5c056e8f62d45ba8215e5cb8f50dfccb198b4b9fbea8500674f3443e4689589"
+  end
 
-    system "./autogen.sh" if build.head?
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/16/d8/bc6316cf98419719bd59c91742194c111b6f2e85abac88e496adefaf7afe/six-1.11.0.tar.gz"
+    sha256 "70e8a77beed4562e7f14fe23a786b54f6296e34344c23bc42f07b15018ff98e9"
+  end
 
+  def install
+    # Reduce memory usage for Circle CI.
+    ENV["MAKEFLAGS"] = "-j8" if ENV["CIRCLECI"]
+    system "./build_boost.sh"
+    system "./autogen.sh"
```

Comment:
Is this line necessary? The formula doesn't currently depend on autoconf or automake.

Reply:
```
./autogen.sh: 8: ./autogen.sh: autoreconf: not found
```
https://circleci.com/api/v1.1/project/github/brewsci/homebrew-bio/1005/output/112/0?file=true
Add…
```ruby
depends_on "autoconf" => :build
depends_on "automake" => :build
```

Reply:
Will do.  I did have this in originally, then took it out again because of some issue with libtool.  Is there a 'depends_on "libtool" => build' option too?